src/dom/dom.js

```diff
@@ -2113,6 +2113,14 @@ p5.Element.prototype.size = function(w, h) {
   if (arguments.length === 0) {
     return { width: this.elt.offsetWidth, height: this.elt.offsetHeight };
   } else {
+    if (this.elt instanceof HTMLCanvasElement) {
+      p5._friendlyError(
+        'Try using the method resizeCanvas for canvas instead of size method check this for reference',
```

Comment:
Is there a way that we could be more direct with this language? `Use resizeCanvas() instead of size() when changing the size of a canvas or graphics object.`?

Reply:
Note that you don't need to take my suggestion verbatim. It is just meant to illustrate what I mean by direct.

Reply:
yeah ok I will make the required changes